_digest/october-2020/choosing-your-major-by-money-or-passion.mdx

```diff
@@ -0,0 +1,32 @@
+  
+  <p>According to bestcolleges.com, when choosing a major, one should consider the following:</p>
+  <li> <u>Overall program cost</u> </li>
+  <li> <u>Salary expectations</u> </li>
+  <li> <u>Employment rates in the field</u> </li>
+  <li> <u>Advanced degree opportunities</u> </li>
+  
+  <p><br>Four simple steps to choose your major. Sounds easy enough, right? While these are good tips to keep in mind, none of them touches on how your major may benefit you mentally. Committing several years of your life to one subject can be emotionally draining. So, it may be beneficial to choose a subject that genuinely interests you and fulfills your desires.</p>
+  <p>When you ask a kid, “What do you want to be when you’re older?” the typical response is probably around the lines of, “I want to be a(n) astronaut/singer/actor, etc.”  Kids tend to have pure desires. Their desires only consist of what will make them happy in the end. It’s not polluted by societal ideals like what would earn them the most money. Eventually, most kids grow out of these dreams and start to accept the reality for what it is – that getting your dream job isn’t as easy as it sounds. I’m not saying it will never happen. It’s just not guaranteed. It could be more reasonable to get a blue or white collar job.</p>
+  <p>I find the word “education” misleading, and here’s why. When you’re a kid, you have all these dreams and goals. However, from a young age, “education” becomes associated with school and how investing time and money into it will reward you with a bright future. But as our schooling career progresses, we develop a love-hate relationship with the standardized system, and as a result, education itself. We understand that devoting all these years to school can help us achieve a successful career, but we slowly begin to question whether it’s worth the consequential emotional and physical stress.</p>
+  <p>According to Google, the word “education” can be split up into two definitions.</p>
+  
+  <ol>
+    <li> <u><strong><em>“</em></strong><strong><em>The process of receiving or giving systematic instruction, especially at a school or university.”</em></strong> This definition is what most of us think of. It’s the definition that was fed to us by our parents and school. The phrase “systematic instruction” has a negative undertone – as if students don’t have control over our learning, which is what it often feels like. The overpowering pressure to do well in a system built to challenge us has led students to stray away from the principles of schooling, and consequently, education as a whole.</u> </li>
+    <li> <u><strong> </strong><strong><em>“An enlightening experience.”</em></strong> I believe this terminology should be reflected more into our schooling. Regardless of what field you end up pursuing, education can, and should be, an illuminating and uplifting experience.</u> </li>
```

Comment:
The first strong opening and closing tag seems wrong. It's redundant and not needed since it's not encompassing anything